sqldelight-compiler/src/test/kotlin/com/squareup/sqldelight/core/queries/ExpressionTest.kt

```diff
@@ -156,6 +156,18 @@ class ExpressionTest {
     ).inOrder()
   }
 
+  @Test fun `sum always returns nullable`() {
```

Comment:
Add `@Ignore("issue link")` and we can merge the test!

Reply:
Okay done!